gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientImpl.java

```diff
@@ -75,6 +77,7 @@ public class GoogleCloudStorageClientImpl extends ForwardingGoogleCloudStorage {
       @Nullable Credentials credentials,
       @Nullable HttpTransport httpTransport,
       @Nullable HttpRequestInitializer httpRequestInitializer,
+      @Nullable List<ClientInterceptor> gRPCInterceptors,
```

Comment:
Who is calling this? I only see this called from the tests? Also expected this to be behind traceenabled flag

Reply:
Why we need this in production?